### DISCUSSION TITLE
There is a proposal to developers to expand the functionality

### DISCUSSION THREAD

**Discussion by weisseskopfadler1** [Weasyl/weasyl#1371] [Ideas]
Here are some suggestions (maybe other users will agree with them too):

1. you need to enter manual user confirmation (in case there are problems with confirmation via e-mail)
2. Ability to appoint and remove administrators, moderators through the admin panel (Can be appointed or removed by the director)
3. add the ability to edit site information through the admin panel, as well as edit site rules (upload policy, etc.)
4. Make adding ads through the admin panel
5. Add the ability to add new languages through the admin panel ​​(make it multilingual)
6. add the ability to create a new director account after installing the site

**Comment by charmander** [comment]
Other than 1 as part of upcoming changes (and possibly 4 if we decide to run ads some day), these aren’t likely to ever be in Weasyl, sorry.
